src/Symfony/Component/Validator/Constraints/Conditional.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\ExpressionLanguage\Expression as ExpressionObject;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\Validator\Exception\LogicException;
+
+/**
+ * @Annotation
+ * @Target({"CLASS", "PROPERTY", "METHOD", "ANNOTATION"})
+ */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class Conditional extends Composite
+{
+    public $condition;
+
+    public $constraints = [];
+
+    public $values = [];
+
+    /**
+     * @param string|ExpressionObject|array $condition The expression to evaluate or an array of options
+     */
+    public function __construct(
```

Comment:
please in one line

Reply:
In my opinion, I think it's easier to read when there is a lot of parameters for a function to have them in multiple lines.
Plus, it's easier to determine the line that has been changed in Git if we only alter one parameter.
I don't know the coding standards in this project, but there are already methods in multiple lines.
Of course, if you really want me to change, I'll do it.

Reply:
I have the impression that there are no standards, on the other validators we find the 2 ways.
Personally, I prefer the multi-line because of the number of parameters.